### PR TITLE
[PW-4912] - Shopware6 unit testing setup

### PR DESCRIPTION
**Pull request by Morerice**
## Summary
1. Require the phpunit dep in Composer
1. Add at least one simple test to verify that the setup works
2. Set up Github Actions to run tests

## Tested scenarios
**Inception**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ~        ######  #####( ######  #####. ######  #####  ######  #####  ######
+  ~ ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+  ~ ###### ######  #####( ######  #####. ######  #####          #####  ######
+  ~ #############  #############  #############  #############  #####  ######
+  ~  ############   ############  #############   ############  #####  ######
+  ~                                      ######
+  ~                               #############
+  ~                               ############
+  ~
+  ~ Adyen PrestaShop plugin
```
Comment (peterojo) [inline phpunit.xml:15]: Adyen Payment Module

File: src/Test/bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
```
Comment (peterojo) [inline src/Test/bootstrap.php:22]:
```suggestion
 * Author: Adyen <shopware@adyen.com>
```

File: phpunit.xml (hunk 1 above)
Comment (Morerice) [reply phpunit.xml]: Updated in 424c657

File: src/Test/bootstrap.php (hunk 2 above)
Comment (Morerice) [reply src/Test/bootstrap.php]: Updated in 424c657

**Review by peterojo — APPROVED** [review]

**Pull request merged**